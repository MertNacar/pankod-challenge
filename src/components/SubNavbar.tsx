import React from "react";

interface IProp {
  title: string;
}

const SubNavBar: React.FC<IProp> = (props) => {
  return (
    <nav className="shadow navbar navbar-dark bg-dark">
      <h4 className="navbar-brand px-5">{props.title}</h4>
    </nav>
  );
};

export default SubNavBar;
