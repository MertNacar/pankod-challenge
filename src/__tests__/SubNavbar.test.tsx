import React from "react";
import { shallow } from "enzyme";
import SubNavbar from "../components/SubNavbar";

describe("SubNavbar Component", () => {
  const wrapper = shallow(<SubNavbar title="Popular Titles" />);

  it("is exists", () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  it("return SubNavbar with Popular Titles field", () => {
    expect(wrapper).toMatchSnapshot("SubNavbar");
  });
});
