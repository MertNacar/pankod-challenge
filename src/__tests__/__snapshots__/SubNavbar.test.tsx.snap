// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubNavbar Component return SubNavbar with Popular Titles field: SubNavbar 1`] = `
<nav
  className="shadow navbar navbar-dark bg-dark"
>
  <h4
    className="navbar-brand px-5"
  >
    Popular Titles
  </h4>
</nav>
`;
